### DISCUSSION TITLE
Hotkey space key marks file and jumps to next file

### DISCUSSION THREAD

**Discussion by toni-mango** [mikekazakov/nimble-commander#451] [General]
Hello,

thx for this great file commander app! Is there a possiblity to configure space-key to mark file and jump directly to the next file (as it was in Norton Commander and other filemanager apps) ??

Best regards
Toni

**Comment by mikekazakov** [comment]
Hi, thank you!

Take a look at https://github.com/mikekazakov/nimble-commander/blob/main/Docs/Help.md#selection.
By default this hotkey is bound to `Enter`, but you can change to `Space` in Settings->Hotkeys.

Michael

**Comment by toni-mango** [comment]
Great! Thank you for your reply! Works great.

Cheers
Toni
